pallets/thea-executor/src/weights.rs

```diff
@@ -0,0 +1,139 @@
+
+//! Autogenerated weights for `thea_executor`
+//!
+//! THIS FILE WAS AUTO-GENERATED USING THE SUBSTRATE BENCHMARK CLI VERSION 4.0.0-dev
+//! DATE: 2023-06-15, STEPS: `10`, REPEAT: `20`, LOW RANGE: `[]`, HIGH RANGE: `[]`
+//! WORST CASE MAP SIZE: `1000000`
+//! HOSTNAME: `Krishnas-MacBook-Pro.local`, CPU: `<UNKNOWN>`
+//! EXECUTION: None, WASM-EXECUTION: Compiled, CHAIN: None, DB CACHE: 1024
+
+// Executed Command:
+// ./target/debug/polkadex-node
```

Comment:
Benchmark is done using debug mode, which is not ideal but can be replaced in our final run today.

Reply:
I'll PR everything separately from dedicated machine when all benchmarks are merged in.